jekyll/_cci2/orb-author.md

```diff
@@ -17,104 +17,125 @@ version:
+    * [Orb Pack Command]({{site.baseurl}}/2.0/orb-concepts/#orb-packing)
+    * [Orb Init Command](https://circleci-public.github.io/circleci-cli/circleci_orb_init.html)
 * [Orb Tools Orb](https://circleci.com/developer/orbs/orb/circleci/orb-tools)
 
 ### Getting started
 {: #getting-started }
 
-**Note:** While the outlined process below only mentions GitHub, the development kit also works with Bitbucket repositories.
-
-To begin creating your new orb with the orb development kit, follow these steps. The starting point is creating a new repository on [GitHub.com](https://github.com).
+To begin creating your new orb with the Orb Development Kit, follow these steps. The starting point is creating a new repository on [GitHub.com](https://github.com).
 
 Ensure the organization on GitHub is the owner for the [CircleCI namespace]({{site.baseurl}}/2.0/orb-concepts/#namespaces) for which you are developing your orb. If this is your own personal organization and namespace, you need not worry.
 
 1. **Create a new [GitHub repository](https://github.com/new).**
-<br>
-The name of your repository is not critical, but we recommend something similar to "myProject-orb". ![Orb Registry]({{site.baseurl}}/assets/img/docs/new_orb_repo_gh.png)
+
+    The name of your repository is not critical, but we recommend something similar to "myProject-orb". ![New GitHub Repo]({{site.baseurl}}/assets/img/docs/new_orb_repo_gh.png)
 
     When complete, you will be brought to a page confirming your new repository and you should see the generated git URL. Write down the git URL, you will need it in step 4. You can select SSH or HTTPS, which ever you can authenticate with. ![Orb Registry]({{site.baseurl}}/assets/img/docs/github_new_quick_setup.png)
 
     **Note:** While you must create a local directory for your orb before initializing, it is not necessary to pull down the orb repository. This process will be completed in the `orb init` process and pulling the repository beforehand will cause issues.
     {: class="alert alert-warning"}
 
-1. **Open a terminal and initialize your new orb project using the `orb init` CLI command.** **If you are using CircleCI server, you should ensure the `--private` flag is used here to keep your orbs private within your installation.**
-<br>
-To initialize a **[public]({{site.baseurl}}/2.0/orb-intro/#public-orbs)** orb:
-<!---->
-```shell
-circleci orb init /path/to/myProject-orb
-```
-To initialize a **[private]({{site.baseurl}}/2.0/orb-intro/#private-orbs)** orb:
-```shell
-circleci orb init /path/to/myProject-orb --private
-```
-<!---->
+1. **Update the CircleCI CLI**
+
+    Ensure you are using the latest version of the CircleCI CLI. You must be using version < CHANGE THIS LATER > or later.
+
+    ```shell
+    $ circleci update
+
+    $ circleci version
+    ```
+
+1. **Initialize your orb**
+
+    Open a terminal and initialize your new orb project using the `orb init` CLI command.
```

Comment:
Is it necessary to open another terminal  - they've just done the CLI update in one the step before?